src/routes/routes.js

```diff
@@ -0,0 +1,97 @@
+import { Router } from 'express';
+import Joi from 'joi';
+import validator from 'express-joi-validation';
+
+import UsersController from '../controllers/controller';
+
+const controller = new UsersController();
+const router = Router();
+const userValidator = validator.createValidator({});
+
+const querySchema = {
+  userInfo: Joi.object({
+    login: Joi.string().required(),
+    password: Joi.string().required(),
+    age: Joi.number()
+      .min(4)
+      .max(130)
+      .required(),
+  }),
+  findUser: Joi.object({
+    id: Joi.string().required(),
+  }),
+};
+
+router
+  .get(
+    '/users/:id',
+    userValidator.params(querySchema.findUser),
+    (req, res) => {
+      const user = controller.getUser(req.params.id);
+
+      if (user) {
+        res.send(user);
+        return;
+      }
+
+      res.status(400).send('A user with this ID was not found.');
+    },
+  )
+  .put(
+    '/users/:id',
+    userValidator.query(querySchema.userInfo),
+    userValidator.params(querySchema.findUser),
+    (req, res) => {
```

Comment:
You can move these functions to separate place - thus code will look more clear

Reply:
Is this in a separate validator controller or something of that sort? What's the best way to approach this?

Reply:
updated to body, kept validator as middleware